### PR TITLE
Support basic templating on default incident fields configuration

### PR DESCRIPTION
**Pull request by louisfelix**
In `servicenow.yml`, the `default_incident` fields configuration now support templating from the alert group data structure.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -319,3 +305,47 @@ func getGroupKey(data template.Data) string {
...
+		incident[key], err = applyTemplate(val.(string), data)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func applyTemplate(text string, data template.Data) (string, error) {
+	tmpl, err := tmpltext.New("").Parse(text)
```
Comment (vincentcreusotfx) [inline main.go]: What about putting a name on this template ?

File: main.go [hunk 2]
```diff
@@ -319,3 +305,47 @@ func getGroupKey(data template.Data) string {
...
+	}
+
+	return result.String(), nil
+}
+
+func validateIncident(incident Incident) error {
+	impact := incident["impact"].(string)
+	if len(impact) > 0 {
+		if _, err := strconv.Atoi(impact); err == nil {
+			log.Errorf("'impact' field value is '%v' but should be an integer. Process was not stopped, but your configuration need to be fixed", impact)
```
Comment (vincentcreusotfx) [inline main.go]: The error message should be "should be a digit" if I understand well ?

File: main.go [hunk 3]
```diff
@@ -319,3 +305,47 @@ func getGroupKey(data template.Data) string {
...
+		}
+	}
+
+	urgency := incident["urgency"].(string)
+	if len(urgency) > 0 {
+		if _, err := strconv.Atoi(urgency); err == nil {
+			log.Errorf("'urgency' field value is '%v' but should be an integer. Process was not stopped, but your configuration need to be fixed", urgency)
```
Comment (vincentcreusotfx) [inline main.go]: Same here

File: main.go (hunk 2 above)
Comment (vincentcreusotfx) [inline main.go]: What is happening then if it continues ?

File: main.go (hunk 1 above)
Comment (louisfelix) [reply main.go]: Done

File: main.go (hunk 2 above)
Comment (louisfelix) [reply main.go]: Clarified in error message.

**Pull request merged**